.github/pull-request-template.md

```diff
@@ -9,16 +9,16 @@ your original PR has been merged into the main branch. Once the cherry-pick PR
 has merged, remove the cherry-pick label from the original PR.
 
 Use one of the new content templates:
-- [Concept](./docs/contributor/templates/template-concept.md) -- Conceptual topics explain how things
+- [Concept](docs/help/contributor/templates/template-concept.md) -- Conceptual topics explain how things
 work or what things mean. They provide helpful context to readers. They do not include procedures.
-- [Procedure](./docs/contributor/templates/template-procedure.md) -- Procedural (how-to) topics
+- [Procedure](docs/help/contributor/templates/template-procedure.md) -- Procedural (how-to) topics
 include detailed steps to perform a task as well as some context about the task.
-- [Troubleshooting](./docs/contributor/templates/template-troubleshooting.md) -- Troubleshooting 
+- [Troubleshooting](docs/help/contributor/templates/template-troubleshooting.md) -- Troubleshooting
 topics list common errors and solutions.
-- [Blog](./docs/contributor/templates/template-blog-entry.md) -- Instructions and a template that you
+- [Blog](docs/help/contributor/templates/template-blog-entry.md) -- Instructions and a template that you
 can use to help you post to the Knative blog.
 
-Consult [Knative contributor's guide](./help/contributing) for all resources for contributing to
+Consult [Knative contributor's guide](./help/contributor/readme.md) for all resources for contributing to
```

Comment:
```suggestion
Consult [Knative contributor's guide](./help/contributor/README.md) for all resources for contributing to
```

Reply:
I made two suggestions above, choose the one you think is best: 
1. Link to the website version of this page (https://knative.dev/docs/help/contributor/) instead of to GitHub 
2. Fix the GitHub link ([./help/contributor/README.md](https://github.com/knative/docs/tree/mkdocs/docs/help/contributor/README.md))  (lowercase "readme" doesn't seem to work).